crates/stackable-operator/crds/ListenerClass.yaml

```diff
@@ -77,16 +77,17 @@ spec:
                 description: Annotations that should be added to the Service object.
                 type: object
               serviceExternalTrafficPolicy:
-                default: Local
                 description: |-
-                  `externalTrafficPolicy` that should be set on the created [`Service`] objects.
+                  Service Internal Traffic Policy enables internal traffic restrictions to only route internal
+                  traffic to endpoints within the node the traffic originated from. The "internal" traffic
+                  here refers to traffic originated from Pods in the current cluster. This can help to reduce
+                  costs and improve performance. See [Kubernetes docs][k8s-docs].
 
-                  The default is `Local` (in contrast to `Cluster`), as we aim to direct traffic to a node running the workload
-                  and we should keep testing that as the primary configuration. Cluster is a fallback option for providers that
-                  break Local mode (IONOS so far).
+                  [k8s-docs]: https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/
```

Comment:
note: This description seems to be completely out-of-date when compared to the doc comment below.

Reply:
Good catch!
Well, that's sadly another kube schema bug we didn't fix yet. It picks the description from
`pub enum KubernetesTrafficPolicy`
instead of 
`pub service_external_traffic_policy: Option<core_v1alpha1::KubernetesTrafficPolicy>`
(which should be picked as it is more concrete)
CC @NickLarsenNZ

As soon as this bug is fixed the description on the field (and not the enum) should show up

Reply:
I guess ideally we might want to try to combine them. But that is more than a "fix".

Reply:
Yeah I'm not blocking this PR if we feel like this fix should be included in 25.11.0. I'll let you decide that.

Reply:
Yes, it definitely should be, thanks!